docs/src/main/sphinx/release/release-419.md

```diff
@@ -19,16 +33,33 @@
 
 * Improve performance of reading ORC files with Bloom filter indexes. ({issue}`17530`)
 * Improve performance of reading Parquet files. ({issue}`17612`)
+* Improve optimized Parquet writer performance for
+  [structural data types](structural-data-types). ({issue}`17665`)
+* Fix query failure when reading ORC files with a shorthand timezone id in the
+  Stripe footer. You can set the `hive.orc.read-legacy-short-zone-id`
+  configuration property to `true` to enable this behavior. ({issue}`12303`)
+* Fix query failure for tables with file paths that contain non-alphanumeric
+  characters. ({issue}`17621`)
 
 ## Hudi connector
 
 * Improve performance of reading Parquet files. ({issue}`17612`)
+* Improve performance when writing Parquet files with
+  [structural data types](structural-data-types). ({issue}`17665`)
 
 ## Iceberg connector
 
+* Add support for the [Nessie catalog](iceberg-nessie-catalog). ({issue}`11701`)
 * Disallow use of the `migrate` table procedure on Hive tables with `array`,
   `map` and `row` types. Previously, this returned incorrect results after the
   migration. ({issue}`17587`)
 * Improve performance of reading ORC files with Bloom filter indexes. ({issue}`17530`)
 * Improve performance of reading Parquet files. ({issue}`17612`)
+* Improve performance when writing Parquet files with
+  [structural data types](structural-data-types). ({issue}`17665`)
 * Improve performance of reading table statistics. ({issue}`16745`)
+
+## SPI
+
+* Fix incorrect behavior when using the `DEFAULT_ON_NULL` return convention. ({issue}`17706`)
```

Comment:
This is a fix to a change that hasn't been released (https://github.com/trinodb/trino/pull/17212), so it doesn't need a release note item.